installation/upgrade-notes.md

```diff
@@ -16,42 +22,63 @@ Users should switch to the `fluent-bit` package.
 
 ## Fluent Bit v1.6
 
-If you are migrating from previous version of Fluent Bit please review the following important changes:
+If you are migrating from previous version of Fluent Bit, review the following
+important changes:
 
 ### Tail Input Plugin
 
-Now by default the plugin follows a file from the end once the service starts \(old behavior was always read from the beginning\). For every file found at start, its followed from it last position, for new files discovered at runtime or rotated, they are read from the beginning.
+By default, the plugin follows a file from the end once the service starts, instead
+of reading from the beginning. Every file found at start is followed from it last
+position. New files discovered at runtime or rotated are read from the beginning.
```

Comment:
```suggestion
By default, the tail input plugin follows a file from the end after the service starts,
instead of reading it from the beginning. Every file found when the plugin starts is
followed from it last position. New files discovered at runtime or rotated are read
from the beginning.
```

Reply:
I don't know what the "rotated" bit is about and couldn't even venture a guess.